scipy/stats/stats.py

```diff
@@ -5718,3 +5720,179 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def rvs_ratio_unif(f, size=1, x1=None, x2=None, y2=None, shift=0, maxiter=None,
+                   bounds=None):
+    """
+    Generate random samples from a probability density function using the
+    ratio-of-uniforms method.
+
+    Given a univariate probability density function (pdf) f and a constant c,
```

Comment:
Please put 5731 to 5746 in the notes section. That's normal for something like this. I'm not requesting that you remove it completely, it just doesn't belong here.
For example [`stats.norm`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html#scipy.stats.norm) describes it's PDF in the notes.